demos/common/cpp/models/src/detection_model_ssd.cpp

```diff
@@ -120,9 +161,20 @@ void ModelSSD::prepareInputsOutputs(InferenceEngine::CNNNetwork& cnnNetwork) {
     // --------------------------- Prepare output blobs -----------------------------------------------------
     slog::info << "Checking that the outputs are as the demo expects" << slog::endl;
     OutputsDataMap outputInfo(cnnNetwork.getOutputsInfo());
-    if (outputInfo.size() != 1) {
-        throw std::logic_error("This demo accepts networks having only one output");
+    if (outputInfo.size() == 1) {
+        prepareSingleOutput(outputInfo);
+        shoulPostprocessMultipleOutputs = false;
+    }
+    else if(outputInfo.size() == 8 || outputInfo.size() == 9) {
```

Comment:
`prepareMultipleOutputs()` and `postprocessMultipleOutputs()` don't care if `outputInfo` has 8 or 9 entries. They care about existence of `boxes` and `labels` and that's what should be checked. Since `prepareMultipleOutputs()` already checks that, this `if` is not needed at all.

Reply:
we still need it as we have to figure out subtype of model - 1-output SSD or 8/9 outputs person detection models

Reply:
Logic is changed in new commit